modules/DMZ/project_config/README.md

```diff
@@ -63,6 +70,61 @@ Here is an example XML file `sample_config.xml` to demonstrate all the uses of a
   </Models>
 </MLTF-Configuration>
 ```
+
+## Required XML elements
+Below is a bare minimum XML example showing what XML elements are **required**.
+```XML
+<MLTF-Configuration>
+  <Files/>
+  <Prediction/>
+  <ID_label/>
+  <Models/>
+</MLTF-Configuration>
+```
+
+## Required XML attributes
+Below is an XML example showing the required attributes for each element.
+```XML
+<MLTF-Configuration
+  name="SE2-KaggleComp"
+  user="asclines" >
+  <Firebase
+    url="FirebaseDatabaseURL"
+    account="FullPathToServiceAccount"/>
+  <Files>
+    <Folder
+      type="test"

```

Comment:
Does this mean that everything in a folder must be of the same type? e.g. I can't intermix test and train data in a single folder?